Library/Formula/sdf2-bundle.rb

```diff
@@ -0,0 +1,37 @@
+class Sdf2Bundle < Formula
+  desc "parser and parser generator for syntax definition formalism (SDF2)"
+  homepage "http://www.program-transformation.org/Sdf/SdfBundle"
+  url "http://releases.strategoxt.org/strategoxt-0.17/sdf2-bundle/sdf2-bundle-2.4pre212034-37nm9z7p/sdf2-bundle-2.4.tar.gz"
```

Comment:
@mikemcquaid: fair. I'm going to see if anyone is still alive upstream to assist me in getting something sane up and running. Don't hold your breath. Thanks as always for your hard work, dedication and feedback.